tests/framework/PostProcessors/TemporalDataMiningPostProcessor/Clustering/test_tdmSKLearnAffinityPropogation.xml

```diff
@@ -145,7 +145,8 @@
 
   <DataObjects>
     <HistorySet name='clusterInfo'>
-      <Output>x1,x2,labels</Output>

```

Comment:
is there a reason why the labels have been removed?

Reply:
I moved the labels from Input to Output in my previous modifications, in this case it is consistent  with output HistorySet, we keep the labels in Output space. However, in the devel, we do keep the labels in the input space for the solutionExport, and in this modification, I moved the labels to the Input space.